lib/libvmod_std/vmod.vcc

```diff
@@ -202,6 +203,15 @@ case, you may need to modify the string, for example with
 
   set beresp.http.served-by = regsub(std.fileread("/etc/hostname"), "\R$", "");
 
+$Function BLOB blobread(PRIV_CALL, STRING)
```

Comment:
I am not so happy with adding another variant of `std.fileread()` which has some relevant drawbacks, see motivation in https://code.uplex.de/uplex-varnish/libvmod-file/blob/master/README.rst
I would rather consider adding something like the above to varnish-cache

Reply:
I agree, but this is something we can get built-in for free by leveraging the `std.fileread()` implementation. This is why I went for this approach, and anything more "serious" can rely on third-party VMODs.